bundles/org.openhab.binding.liquidcheck/src/main/resources/OH-INF/i18n/liquidcheck_de.properties

```diff
@@ -0,0 +1,43 @@
+# binding
```

Comment:
I think you need to delete this file. Only the default (english) file should be provided. Any additional translation should be made from crowdin.

Reply:
File has been deleted.